docs/modules/ROOT/pages/api/account.adoc

```diff
@@ -0,0 +1,213 @@
+:github-icon: pass:[<svg class="icon"><use href="#github-icon"/></svg>]
+:snip6: https://github.com/ericnordelo/SNIPs/blob/feat/standard-account/SNIPS/snip-6.md[SNIP-6]
+
+= Account
```

Comment:
i think it's worth documenting the `Call` struct, don't you think? not only it's very useful to use/study the library, but it's also referenced multiple times in this very section